notebooks/official/tabular_workflows/tabnet_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1134 @@
+{
```

Comment:
<p>I would split up these definitions so that we don't have to scroll up and down between the code and the definitions. data_source_csv_filenames and data_source_bigquery_table_path can stay here, but most of the other definitions would be better located in "Setup training configuration". dataflow_subnetwork could go to the VPC related config section. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/726/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>Done.</p>